index.html

```diff
@@ -1,21 +1,325 @@
+                            <span></span>
+                        </p>
+                        <p class="paragraphtext1">
+                            <span></span>
+                        </p>
+                    </div>
+                       
+                    <div class="moveup">
+                        <figure>
+                            <img src="img/red-bottles.webp" class="BottlesPicture"
+                                alt="a display of a melon and bottled red juice">
+                        </figure>
+                        <p class="subheader2format text">
+                            <span>Soft & Creamy</span>
+                            <span>Organic Smoothies</span>
+                            <span></span>
+                        </p>
+                        <p class="paragraphtext1">
+                            <span></span>
+                        </p>
+                    </div>                        
+                    <div>
+                        <figure>
+                            <img src="img/green-bottles.webp" class="BottlesPicture" alt="2 bottles of green juice">
+                        </figure>
+                        <p class="subheader2format text">
+                            <span>The Power of</span>
+                            <span>Wellness Shots</span>
+                            <span></span>
+                        </p>
+                        <p class="paragraphtext1">
+                            <span></span>
+                        </p>
+                    </div>
+                </div>
+
+            </article>
+            <article class="ViewAllButton">
+                <div class="centre">
+                    <a href="viewing.html" class="AButton LButton">View All</a>
+                </div>
+            </article>
+        </article>
+
+        <article class="WhiteArea">
+            <div class="row-flexdisplay">
+                <figure>
+                    <img src="img/boxof-bottles.webp" class="WhitePicture" alt="a box of bottled assorted juices">
+                </figure>
+                <div>
+                    <p class="subheader2format text WhiteHeader">
+                        <span>Discover Our</span>
+                        <span>Special Cleanse Kit</span>
+                        <span></span>
+                    </p>
+                    <p class="paragraphtext2">
+                        <span></span>
+                    </p>
+                    <article class="TryItButton">
+                        <div class="centre">
+                            <a href="#" class="AButton WButton">Try It</a>
+                        </div>
+                </div>
+            </div>
+        </article>
+
+        <p class="subheader2format text GetJuiceBarHeader">
+            <span>Get Juice Bar</span>
+            <span></span>
+        </p>
+
+        <article class="display-3items-flexrow">
```

Comment:
Somehow, this section has a white right gap in the desktop version. The right way is to add the background color to the `display-3items-flexrow` class, instead of the `display-3numbers` class